codestyle/druid-forbidden-apis.txt

```diff
@@ -42,4 +42,7 @@ com.ibm.icu.text.SimpleDateFormat#<init>(java.lang.String)
 
 @defaultMessage For performance reasons, use the utf8Base64 / encodeBase64 / encodeBase64String / decodeBase64 / decodeBase64String methods in StringUtils
 org.apache.commons.codec.binary.Base64
-com.google.common.io.BaseEncoding.base64
\ No newline at end of file
+com.google.common.io.BaseEncoding.base64
+
+@defaultMessage Use com.google.errorprone.annotations.concurrent.GuardedBy
```

Comment:
It could be just a single line: `javax.annotations.concurrent.GuardedBy @ Use com.google.errorprone.annotations.concurrent.GuardedBy`. Please keep in mind when adding forbidden API rules in the future.